proposed/version-support-meta.md

```diff
@@ -0,0 +1,130 @@
+there is no incentive for public hosts to offer newer PHP versions that would 
+resolve that issue.
+
+This standard draws inspiration from the GoPHP5 initiative from 2007. In that 
+case, numerous projects and hosts collectively agreed on a drop-dead date for
+all PHP 4 versions, and that future releases would standardize on at least PHP 5.2.
+That initiative was successful in breaking the log-jam that kept PHP 4 alive well
+past its retirement and delayed the implementation of PHP 5.
+
+Rather than periodic "surge" efforts, this standard creates a rolling-GoPHP process
+that both projects and hosts can build into their planning.
+
+
+3. Scope
+--------
+
+## 3.1 Goals
+
+This standard serves as a "community covenant" for projects and hosts to agree on
+an approximate schedule for when they will adopt and drop support for PHP versions.
+By synchronizing release schedules, no project is overly penalized for being first
+to drop support for a given PHP version and public hosts have a clear timeline
+they can factor into their normal upgrade schedules.
+
+## 3.2 Non-Goals
+
+This document does not seek to impose any architectural decisions on participating
+projects, nor to dictate what language features they use.
+
+
+4. Example release schedule
+---------------------------
+
+The following examples help illustrate with examples how a participating project
+and host could operate with respect to PHP releases.
+
+## 4.1 For projects
+
+### 4.1.1 PHP minor releases
+
+Flubber 4.2 is the current release.  PHP 9.2.3 is the current stable release.
+
+On 1 March 2020, PHP 9.3.0 is released.
+
+Flubber 4.3 is released on 20 March. It is unaffected, because PHP 9.3 has been
+out for less than 90 days.
+
+Flubber 4.4 is released on 14 July. It is required to support PHP 9.3 because
+it has been out for more than 90 days.
+
+On 1 March 2021, PHP 9.4. is released.
+
+Flubber 5.0 is released on 20 March. It is unaffected, because PHP 9.4 has been
+out for less than 6 months.
+
+### 4.1.2 PHP major releases
+
+
+## 4.2 For hosts
+
+PHP 9.2.3 is the current stable release, and was released on 1 March 2020.
+PHP 9.1 is the legacy release.  PHP  9.0 is in security-only status.  Flubber
+Hosting offers PHP 9.2.1.
+
+Flubber Hosting has until 1 June 2020 to offer PHP 9.2.3.  When it does so it
+replaces all its 9.2.1 offerings with 9.2.3.
+
+PHP 9.3.0 is released on 1 July 2020.  Flubber Hosting has until 1 January 2021
+to offer 9.3.0.  It may make it the default for new accounts or not, but both
+new and old accounts must have the option to use it.
+
+PHP 9.0 security support ends entirely on 1 October 2020.  Flubber Hosting has
+until 1 October 2022 to cease offering PHP 9.0.x.
+
+PHP 10.0 is released on 1 April 2021.  Flubber Hosting has until 1 April 2022
+to offer it for both new and old accounts. It may be the default or not.  If
+a new minor or patch release comes out before then, it should be treated
+accordingly.
+
+
+5. Participating projects
```

Comment:
Do we really want to keep a list of participating projects/hosts within the meta document? I think this would be better on the website or something if we want to centralise the keeping of a list but either way it adds a burden of ensuring that 'participating projects' are keeping to it on the FIG.

Maybe it might be best to leave it up to a 3rd party to maintain a list or, as with other PSRs, projects/hosts simply declare compliance with the standard.